docsrc/admin/git.md

```diff
@@ -0,0 +1,52 @@
+# Configuring Git
+
+We currently use [bitbucket.org](https://bitbucket.org) as our cloud git repository.
+
+The person installing and maintaining the Data Library should have or [create an account](https://id.atlassian.com/signup?application=bitbucket&continue=https%3A%2F%2Fbitbucket.org%2Faccount%2Fsignin%2F%3Fnext%3D%252F%26redirectCount%3D1) with [https://bitbucket.org](https://bitbucket.org).
+
+Once you have an account inform the [Data Library staff](mailto:help@iri.coplumbia.edu) to grant you access to the repositories.
+
+## Configure DL Account
+
+* Create a new SSH Key on the Data Library Server
+
+      ssh-keygen -t ed25519 -b 4096 -f ~/.ssh/id_bitbucket
+
+  _This will create an ssh key for use with bitbucket in ~/.ssh/id_bitbucket. Set a good passphrase different
+  from your password. You will need to use this passphrase every time you need to commit changes to the bitbucket repository,
```

Comment:
Is it really important that it be different from the password? Fewer passwords to remember seems better.

The point will be moot if we switch to the OAuth mechanism, which should be possible here since these docs are now for CentOS 9. Please give that a try.